test/utils/file-utils.test.js

```diff
@@ -0,0 +1,110 @@
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+
+const fileUtils = require('../../utils/file-utils');
+
+describe('file-utils', () => {
+  describe('#isUrl', () => {
+    it('should return true if valid url', () => {
+      assert(fileUtils.isUrl('https://google.com'));
+    });
+
+    it('should return false if not valid url', () => {
+      assert.equal(false, fileUtils.isUrl('/path/to/image'));
```

Comment:
This should be the other way around `assert.equal(fileUtils.isUrl('/path/to/image'), false);`